keps/sig-node/127-user-namespaces/README.md

```diff
@@ -551,16 +579,21 @@ use container runtime versions that have the needed changes.
 
 ##### Beta
 
-- Make plans on whether, when, and how to enable by default
+- Gather and address feedback from the community
+- Add API Server feature flag to integrate into [Pod Security Standards (PSS)](#pod-security-standards-pss-integration)
+- Get review from VM container runtimes maintainers
+
+###### Open Questions
+
 - Should we reconsider making the mappings smaller by default?
 - Should we allow any way for users to for "more" IDs mapped? If yes, how many more and how?
 - Should we allow the user to ask for specific mappings?
-- Get review from VM container runtimes maintainers
-- Gather and address feedback from the community
 
 ##### GA
 
 - Gather and address feedback from the community
+- Support stateful pods
```

Comment:
I would caution against doing this in beta -> GA migration. It is like a new functionality that would benefit from being tested as beta or even alpha for a release or two.

Reply:
Removed it.